mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -35,6 +35,8 @@ import org.apache.spark.sql.DataFrame
  * for classification.
  * It supports both binary and multiclass labels, as well as both continuous and categorical
  * features.
+ *
+ * @since 1.4.0
```

Comment:
We should use "@Since" instead of "@since" since we made "@Since" for [inheriting scaladocs](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/annotation/Since.scala)